Makefile

```diff
@@ -12,18 +12,27 @@ IDENTIFIER=$(GOOS)-$(GOARCH)
 help:  ## Show available options with this Makefile
 	@grep -F -h "##" $(MAKEFILE_LIST) | grep -v grep | awk 'BEGIN { FS = ":.*?##" }; { printf "%-15s  %s\n", $$1,$$2 }'
 
+web:
+	@cd dashboard/web && yarn build && cp -a ./dist ../../web
+
+yarn:
```

Comment:
We need to add `.PHONY: yarn` to tell Makefile that it will not try to find the file/directory named `yarn`. See `build` below for example